docs/blog/september-contributions-showcase.md

```diff
@@ -0,0 +1,120 @@
+    - property: og:url
+      content: https://e18e.dev/blog/september-contributions-showcase
+  - - meta
+    - property: og:description
+      content: A showcase of e18e-focused contributions from September
+---
+
+# September contributions
+
+_October 3, 2024_
+
+It's that time of month again where we show off some of the great contributions we've seen around the e18e community in the past month!
+
+# Libraries
+
+This month has been a great one for improvements to existing and upcoming libraries. Many have been listening closely to the e18e community and pushing forward together.
+
+## chokidar 4.x
+
+[chokidar](https://github.com/paulmillr/chokidar) is a library for watching for changes on a file system. Built on top of node's own watcher capabilities, it provides a higher level and simpler interface.
+
+Almost all of us depend on this library in some way, often through the toolchain or framework we use.
+
+Paul ([@paulmillr](https://x.com/paulmillr/), the author of chokidar) and I ([@43081j](https://x.com/43081j)) have been working hard to get the next major version of it over the line for a long time now (the branch was made in 2021!).
+
+Recently, we finally released [4.0.0](https://github.com/paulmillr/chokidar/releases/tag/4.0.0)! :partying_face: this massively reduces the size and complexity.
+
+Those of you who remember the `is-number` fiasco on social media will now also be super happy that we (chokidar) were the largest consumer of it and have since dropped it entirely :tada:
+
+Big wins for the e18e community!
+
+Thanks to [@benjaminmccann](https://x.com/benjaminmccann) and talentlessguy for contributing _many_ v3 to v4 upgrades already!
+
+## fdir / tinyglobby
+
+`fdir` was in last month's showcase but has earned another mention thanks to the author ([thecodrr](https://x.com/thecodrr)) landing various community contributed PRs and features.
+
+This is a great example of where an author has joined the community and worked closely with it to further their library, and provide some much needed fixes and features.
+
+In version [6.4.0](https://github.com/thecodrr/fdir/releases/tag/v6.4.0), we saw two important changes:
+
+- Ability to exclude symlinks
+- Ability to bring your own glob library (e.g. [zeptomatch](https://github.com/fabiospampinato/zeptomatch))
+
+These two changes have helped the community migrate even more high level tools and frameworks across.
+
+Similarly, tinyglobby has shipped a few new versions. Particularly it is now able to determine if a string is a glob or not. This is super useful for migrating more projects to it.
+
+Thanks to pralkarz, [@benjaminmccann](https://x.com/benjaminmccann) and [@superchupudev](https://x.com/superchupu) for already migrating so many projects to these two libraries!
+
+## tinyexec migrations
+
+The community has been working wonders migrating countless projects from `execa` and other heavyweight libraries to the super lean [tinyexec](https://github.com/tinylibs/tinyexec/).
```

Comment:
Should we try to avoid calling other libraries heavyweight to avoid any sensitivity there?
```suggestion
The community has been working wonders migrating countless projects from `execa`, `ez-spawn`, and other libraries to the super lean [tinyexec](https://github.com/tinylibs/tinyexec/).
```

Reply:
You're right, I mostly meant that are "full fat" (in that they have all the extra features you might not usually need) 

But I prefer your wording 👍